mix.exs

```diff
@@ -47,23 +47,12 @@ defmodule EpochElixir.Mixfile do
     [
       {:credo, "~> 0.9.3", only: [:dev, :test], runtime: false},
       {:dialyxir, "~> 0.5", only: [:dev, :test], runtime: false},
-      {:sha3, [github: "szktty/erlang-sha3", ref: "dbdfd12", manager: :rebar]},
-      {:idna, [github: "aeternity/erlang-idna", ref: "24bf647", manager: :rebar, override: true]},
-      {:gen_state_machine, "~> 2.0.1"},
-      {:logger_file_backend, "~> 0.0.10"},
-      {:excoveralls, "~> 0.8.1", only: :test},
-      {:erl_base58, "~> 0.0.1"},
-      {:sext, [github: "uwiger/sext", tag: "1.4.1", manager: :rebar, override: true]},
-      {:edown, "~> 0.8", override: true},
-      {:enacl, github: "aeternity/enacl", ref: "2f50ba6", override: true},
-      {:enoise, github: "aeternity/enoise", ref: "6d793b711854a02d56c68d9959e1525389464c87"},
+      {:idna, [git: "https://github.com/aeternity/erlang-idna", ref: "24bf647", manager: :rebar, override: true]},
```

Comment:
there dependencies that are not needed for the umbrella

Reply:
@thepiwo Which exactly? I left in the general mix.exs only those that are used in all of the apps, and also I had some problems with ranch and idna, because if I put them to the corresponding app, the mix gets these dependencies not from the github but from the hex, I don't know why it's happening, investigating it

Reply:
idna, erl_base58, and maybe edown

Reply:
@thepiwo edown is used for the erlang markdown, I am not sure if we need it at all, but it's probably a general dependency